changelog/7758.bugfix.2.rst

```diff
@@ -0,0 +1 @@
+Fix `sunpy.physics.differential_rotation.differential_rotate` to update the ``reference_date`` attribute of the input map instead of the ``date``.
```

Comment:
```suggestion
Fix :func:`sunpy.physics.differential_rotation.differential_rotate` to update the ``reference_date`` attribute of the input map instead of the ``date``.
```